api/v1alpha1/etcd.go

```diff
@@ -159,6 +159,9 @@ type BackupSpec struct {
 	// All full snapshots beyond this limit will be garbage collected.
 	// +optional
 	MaxBackupsLimitBasedGC *int32 `json:"maxBackupsLimitBasedGC,omitempty"`
+	// FullSnapshotLeaseUpdateInterval defines the interval for retrying to update the full snapshot lease.
+	// +optional
+	FullSnapshotLeaseUpdateInterval *metav1.Duration `json:"fullSnapshotLeaseUpdateInterval,omitempty"`
```

Comment:
Why is a change in API required, especially when we are no longer going to use snapshot leases when we work on steward. Introducing something now in the API which is anyways going to be removed is not so nice.
Also the original issue was to retry update of full snapshot lease from backup-restore if the first attempt failed. Why does this now result in a ETCD API change?